cdip_admin/integrations/management/commands/inreach_migration_script.py

```diff
@@ -0,0 +1,395 @@
+
+                            # Remove the /api/v1.0 part from the base URL
+                            er_site = er_site.replace(
+                                "/api/v1.0", ""
+                            )
+
+                            destination_integration, created = Integration.objects.get_or_create(
+                                type=destination_integration_type,
+                                owner=destination_owner,
+                                base_url=er_site,
+                                name=f"{destination_owner.name} - Earthranger ({er_site}) (Inbound: {inbound.name})"
+                            )
+                            if created:
+                                inreach_er_destination_integration_created += 1
+
+                                # Create AUTH action config for the destination integration (ER)
+                                try:
+                                    er_auth_action = IntegrationAction.objects.get(
+                                        type=IntegrationAction.ActionTypes.AUTHENTICATION,
+                                        name="Auth",
+                                        value="auth",
+                                        description="Earth Ranger Auth action",
+                                        integration_type=destination_integration_type
+                                    )
+                                except IntegrationAction.DoesNotExist:
+                                    self.stdout.write(
+                                        f" -- ER destination auth action does not exist in Gundi v2 -- ")
+                                    return
+
+                                er_auth_config, created = IntegrationConfiguration.objects.get_or_create(
+                                    integration=destination_integration,
+                                    action=er_auth_action,
+                                    data={"token": er_token, "authentication_type": "token"}
+                                )
+                                if created:
+                                    self.stdout.write(f" -- Created new configuration for action '{er_auth_action.name}' for destination integration: {destination_integration.name} (ID: {destination_integration.id})")
+
+                                self.stdout.write(f" -- Created new integration: {destination_integration.name} (ID: {destination_integration.id}) for destination: {er_site}")
+
+                            integration.default_route.destinations.add(destination_integration)
+
+                            # add legacy provider_key field mapping
+                            inbound_field_mapping = copy.deepcopy(DEFAULT_FIELD_MAPPING)
+                            inbound_field_mapping["default"] = er_source_provider
+
+                            field_mappings[str(integration.id)]["obv"][str(destination_integration.id)] = inbound_field_mapping
```

Comment:
Here you'll want to also add:

```suggestion
                            field_mappings[str(integration.id)]["obv"][str(destination_integration.id)] = inbound_field_mapping
                                                        field_mappings[str(integration.id)]["txt"][str(destination_integration.id)] = inbound_field_mapping

```